packages/salesforcedx-utils-vscode/src/commands/commandletExecutors.ts

```diff
@@ -220,13 +224,15 @@ export abstract class LibraryCommandletExecutor<T>
         properties,
         measurements
       );
+      LibraryCommandletExecutor.libraryCommandCompletionEventEmitter.fire(!!success);
```

Comment:
Not sure if it is meaningful. LibraryCommandletExecutor is an abstract class to be extended by other real executors such as `orgLogoutDefault` or `orgLoginAccessToken`. How does the subscriber know the result from which command it is listening to with only a boolean value? Can you help me understand that? @angelamuliu @peternhale

Reply:
Great catch @mingxuanzhangsfdx, there is no way to know. If two commands are run concurrently and are listening for the completion event, knowing which event belongs any given listener is indeterminant.